### PR TITLE
Update azure-mgmt-sql to version 0.15.0 which contains changed signature for operations get/delete/create_or_update in managed_instance_administrators.

### PR DESCRIPTION
**Pull request by SanjaMalesevic**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -51,6 +51,10 @@ Release History
 * Upgrading SDK Version to 0.6.0
 * Add billingplan details info after calling Get-Gatalogs
 
+**SQL**
+
+* Update azure-mgmt-sql to version 0.15.0 which contains changed signature for operations get/delete/create_or_update in managed_instance_administrators.
```
Comment (jaredmoo) [inline src/azure-cli/HISTORY.rst]: This is an internal change so I think it doesn't need to be announced here.

Comment (SanjaMalesevic) [reply src/azure-cli/HISTORY.rst]: It is removed now.

**Comment by yonzhan** [comment]
@Juliehzl  please help review.

File: src/azure-cli/HISTORY.rst (hunk 1 above)
Comment (MyronFanQiu) [reply src/azure-cli/HISTORY.rst]: @jaredmoo @SanjaMalesevic Hello. User should know the change of dependency. We can keep "Update azure-mgmt-sql to version 0.15.0" without "which contains changed signature for operations get/delete/create_or_update in managed_instance_administrators."

**Comment by MyronFanQiu** [comment]
@SanjaMalesevic Kindly ping. Please update the history and I can merge this PR.

**Comment by SanjaMalesevic** [comment]
> @SanjaMalesevic Kindly ping. Please update the history and I can merge this PR.

Updated.

**Pull request merged**